cmake/version_config.hpp.in

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#define CUCO_VERSION_MAJOR @CUCO_VERSION_MAJOR@
+#define CUCO_VERSION_MINOR @CUCO_VERSION_MINOR@
+#define CUCO_VERSION_PATCH @CUCO_VERSION_PATCH@
```

Comment:
Where are these coming from? cuco doesn't have any version numbers

Reply:
They're automatically defined by CMake and populated from the `project(<name> VERSION <ver>)` call. But I'm getting rid of this in favor of `rapids_cmake_write_version_file()`